staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/generic/plugin.go

```diff
@@ -181,7 +182,14 @@ func (c *Plugin[H]) ValidateInitialization() error {
 	go func() {
 		err := c.source.Run(pluginContext)
 		if err != nil && !errors.Is(err, context.Canceled) {
-			utilruntime.HandleError(fmt.Errorf("policy source context unexpectedly closed: %v", err))
+			utilruntime.HandleError(fmt.Errorf("policy source context unexpectedly closed: %w", err))
+		}
+	}()
+
+	go func() {
+		err := c.dispatcher.Run(pluginContext)
```

Comment:
you write elsewhere that `Run` does not block. Then we don't need this go routine.

Reply:
I've removed this go routine.  I can't find any reason to keep it.

cc @alexzielenski